src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -1145,6 +1145,7 @@ private void RefreshViewOnSelectionChange()
             ShowHideAllGeometryPreviewCommand.RaiseCanExecuteChanged();
             SetArgumentLacingCommand.RaiseCanExecuteChanged();
             RaisePropertyChanged("HasSelection");
+            RaisePropertyChanged("IsGeometryOperationEnabled");
```

Comment:
Should we not also do a `RaisePropertyChanged("IsGeometryOperationEnabled")` when we switch to a custom node workspace?